packages/cms-lib/themes.js

```diff
@@ -0,0 +1,140 @@
+const fs = require('fs-extra');
+const path = require('path');
+const os = require('os');
+const { promisify } = require('util');
+
+const request = require('request-promise-native');
+const extract = promisify(require('extract-zip'));
+
+const { logger } = require('./logger');
+const {
+  logFileSystemErrorInstance,
+  logErrorInstance,
+} = require('./errorHandlers');
+
+// TODO: When the boilerplate project cuts a release, use latest instead.
+// https://help.github.com/en/github/administering-a-repository/linking-to-releases#linking-to-the-latest-release
+const THEME_BOILERPLATE_ZIP_URI =
+  'https://github.com/HubSpot/cms-theme-boilerplate/archive/master.zip';
+const ZIP_CONTENT_TYPE = 'application/zip';
+const TMP_BOILERPLATE_FOLDER_PREFIX = 'hubspot-cms-theme-boilerplate-';
+
+/**
+ * @returns {Buffer} Zip data buffer
+ */
+async function downloadCmsThemeBoilerplate() {
+  try {
+    logger.log('Fetching theme...');
+    const zip = await request.get(THEME_BOILERPLATE_ZIP_URI, {
```

Comment:
To be consistent here, can we import and use `cms-lib/http.js`'s `.get`?

Reply:
That encapsulates HubSpot APIs with auth to make that process easier.
Here we're just downloading a zip from a public URI.